setup.py

```diff
@@ -106,6 +106,7 @@
             "dbt-databricks",
             "dbt-redshift",
             "dbt-sqlserver>=1.7.0",
+            "dbt-fabric!=1.8.8",  # Broken API, https://github.com/microsoft/dbt-fabric/compare/v1.8.7...v1.8.8 L183
```

Comment:
Could this change be causing the build failure?

Reply:
Yep looks like dbt-sqlserver has published a release to account for the breaking change in dbt-fabric